danskernesdigitalebibliotek/dpl-cms
# Demonstration af ddb react i dpl cms setup ddfdpdel 85

## What is happening in this PR?
A demonstartion of how to pull ddb-react components and render them is presented

## How to test?
* Run: `task dev:setup`
* If you are using dory, make sure that `dory run` has been executed
* Attach to container: `docker-compose exec cli bash`
* Enable react demo module: `drush en -y ddb_react_demo`
* Navigate to http://dpl-cms.docker/react-ddb-demo and verify that the components are being rendered
